lib/Driver/Driver.cpp

```diff
@@ -2031,6 +2031,12 @@ const ToolChain *Driver::getToolChain(const ArgList &Args) const {
       TC = new toolchains::Darwin(*this, Target);
       break;
     case llvm::Triple::Linux:
+      if (Target.isAndroid()) {
```

Comment:
We're using it for FreeBSD too, though. (Or we probably could be; didn't actually check.) I think it's fair to say Cygwin is a Unix-like *toolchain* even if it's not a Unix-like *OS.*